brands/amenity/pub.json

```diff
@@ -6,5 +6,25 @@
       "brand": "Ezītis miglā",
       "name": "Ezītis miglā"
     }
+  },
+  "amenity/pub|Wetherspoons": {
+    "countryCodes": ["gb"],
+    "tags": {
+      "amenity": "pub",
+      "brand": "Wetherspoons",
+      "brand:wikidata": "Q6109362",
+      "brand:wikipedia": "en:J D Wetherspoon",
```

Comment:
looks like the actual Wikipedia page for this one should be `"en:Wetherspoons"`

Reply:
done

Reply:
It looks like you set it to `"en:Wetherspoon"` instead of the correct  `"en:Wetherspoons"`.

Can you try again?  Please pay attention to the details..

Reply:
done, sorry about that, everything should work fine now